test/frontend-integration-test/helloworld.spec.js

```diff
@@ -99,22 +106,14 @@ describe('deploy helloworld sample run', () => {
       reverse: true,
     });
 
-    await browser.keys('Tab');
+    await $('#runNameInput').click();
     await browser.keys(runName);
 
-    await browser.keys('Tab');
+    await $('#descriptionInput').click();
     await browser.keys(runDescription);
-
-    // Skip over "choose experiment" button
-    await browser.keys('Tab');
-    // Skip over service account help button
-    await browser.keys('Tab');
-    // Skip over "service account" textbox
-    await browser.keys('Tab');
-    // Skip over "Run Type" radio button
-    await browser.keys('Tab');
-
-    await browser.keys('Tab');
+    
+    await $('input#newRunPipelineParam0').click();
```

Comment:
NIT: We can add a comment to explain what we expect this parameter name is for, for example: This is pipeline parameter `message`.